data/mods/Magiclysm/Spells/attunements/Wither_Mage.json

```diff
@@ -52,5 +52,46 @@
     "base_energy_cost": 1200,
     "difficulty": 8,
     "extra_effects": [ { "id": "assassin_toxic_smoke", "hit_self": true }, { "id": "assassin_teleport" } ]
+  },
+  {
+    "type": "SPELL",
+    "id": "life_drain",
+    "name": "Life Drain",
+    "description": "This spell takes some of a target's life; part of it comes to you in a less-potent form.",
+    "effect": "attack",
+    "shape": "blast",
+    "valid_targets": [ "hostile", "ally" ],
+    "flags": [ "LOUD", "VERBAL", "NO_LEGS", "NO_PROJECTILE" ],
+    "min_damage": 15,
+    "max_damage": 65,
+    "damage_increment": 1.5,
+    "min_range": 4,
+    "max_range": 16,
+    "range_increment": 0.35,
+    "damage_type": "cold",
+    "max_level": 35,
+    "spell_class": "WITHER_MAGE",
+    "base_casting_time": 300,
+    "energy_source": "MANA",
+    "base_energy_cost": 550,
+    "final_energy_cost": 250,
+    "energy_increment": -8.6,
+    "difficulty": 10,
```

Comment:
the difficulty for this spell seems a bit high for its effect don't you think?

Reply:
okay this one is really too high, my bad